app/core/custom_app.py

```diff
@@ -61,3 +45,25 @@ def create_configuration_token(
         "iat": datetime.timestamp(issued_at.replace(microsecond=0)),  # No microseconds
     }
     return jwt.encode(to_encode, SECRET_KEY, algorithm=ALGORITHM)
+
+
+def decode_configuration_token(token: str) -> list:
+    """Decodes a configuration token.
+
+    :param str  : A token that has been encoded a list of allowed corpora.
```

Comment:
We haven't just encoded a list of corpora IDs -- also an expiry date and issued at date. We might want to update the param docstring to be more representative of this :)